frontend-project/src/components/Table/DocumentTypeName.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC, useEffect, useState } from 'react';
+import { Spin } from 'antd';
+
+import { fetchDocumentType } from '@/services/documentType';
+
+export const DocumentTypeName: FC<{ id: number }> = props => {
+  const [name, setName] = useState('');
```

Comment:
Tu, jak rozumiem, będziemy to pchać w końcu do reduxa @kuskoman ?

Reply:
@magul tak, szedłbym w kierunku wykonywania prawie wszystkich fetchów z poziomu store. nawet jak nie mają być użyte ponownie, to store jest IMO łatwiejszy do przetestowania, niż użycie serwisu w komponencie